composer.json

```diff
@@ -23,5 +23,8 @@
         "mozilla/wp-sw-manager": "dev-master",
         "marco-c/wp_serve_file": "dev-master"
     },
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "autoload": {
+        "classmap": [ "vendor/mozilla/wp-sw-manager/" ]
```

Comment:
https://getcomposer.org/doc/04-schema.md#classmap

This has to be done when the class doesn't support PSR-4.